archlinux/PKGBUILD-arm64

```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jai-JAP <parjailu@gmail.com>
+# Contributor: Sebastien Chevalier <sebastien.chev@gmail.com>
```

Comment:
Can you put my box86 address instead: ptitseb@box86.org

Reply:
Done ...
Take a look